pkg/controllers/multicluster/clusterchecker/bidirection.go

```diff
@@ -32,7 +32,7 @@ type Bidirection struct {
 }
 
 func (b *Bidirection) Check(clusterManager ctrl.Manager, opts ...Option) CheckResult {
-	remoteClusterList, err := utils.ListRemoteClusters(clusterManager.GetAPIReader())
```

Comment:
I don't think `GetClient` can work here, because checker is always working, but manager may not start.

Reply:
Fixed.